css/citationStyleLanguagePlugin.css

```diff
@@ -0,0 +1,103 @@
+.citation_display .value
+{
+  font-size: 0.75rem;
+}
+
+.citation_display .csl-left-margin
+{
+  display: none;
+}
+
+.citation_display [aria-hidden="true"]
+{
+  display: none;
+}
+
+.citation_display .citation_formats
+{
+  margin-top: 1em;
+  border: 1px solid rgba(0, 0, 0, 0.4);
+  border-radius: 2px;
+}
+
+.citation_display .citation_formats_button
+{
+  position: relative;
+  background: transparent;
+  border: none;
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+  box-shadow: none;
+  padding: 0 1em;
+  width: 100%;
+  font-family: "Noto Sans", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen-Sans", "Ubuntu", "Cantarell", "Helvetica Neue", sans-serif;
```

Comment:
I would prefer option one, because then the syle for the plugin is in the plugin. I've made some changes in the css file, so there shouldn't be problems with non-default themes any more. Only font size for the names of the citation styles in the dropdown and neccessary instructions for the arrow by FontAwesome are set by the plugin, now. Color instructions are also only set for some borders of dropdown.